### ISSUE TITLE
IconGenie v.2.3.0 "skipTrim" param in icongenie profile is being ignored/overwritten

### ISSUE THREAD

**Issue by taliator** [quasarframework/quasar#7440]
**Describe the bug**
"skipTrim" param in icongenie profile is being ignored/overwritten

**Codepen/jsFiddle/Codesandbox (required)**
N/A, only requires an icongenie profile (generated on the command line as per below)

**To Reproduce**
1. Generate an icongenie profile (replacing filenames as appropriate)
    `$ icongenie profile -o foo -a spa,capacitor -i app-icon.png -b app-splashscreen.png --quality 2 --skip-trim`
2. See that profile has "skipTrim: true"
    
![image](https://user-images.githubusercontent.com/1482924/87694775-9b05f900-c75c-11ea-97ec-93fca3beddff.png)

3. Run icongenie using the profile
    `$ icongenie generate -p ./icongenie-foo.json`
4. View in the console that icon trimming is `true` (should be `false` here)
    
![image](https://user-images.githubusercontent.com/1482924/87694160-d6ec8e80-c75b-11ea-88f7-6e7dfdb31751.png)

5. View the generated icons and see that they are trimmed

**Expected behavior**
The icons should not be trimmed when `skipTrim` is `true`

**Screenshots**
See above

**Platform (please complete the following information):**
OS: MacOS Catalina 0.15.5
Node: v14.2.0
NPM: 6.14.4

**Additional context**
I believe this is due to reversed arguments in the [`mergeObjects` call in `generate.js`](https://github.com/quasarframework/quasar/blob/bd35cdffd6f5a5b4c1be33f4ba1d587b39da1379/icongenie/lib/cmd/generate.js#L138)

**Comment by NicolasReibnitz** [comment]
> I believe this is due to reversed arguments in the mergeObjects call in generate.js

It's not necessarily reversed. It depends on what you would want to prioritize, profile file, or command argument. If you would switch the arguments, you could never overwrite the skipTrim option via command argument. But I agree that we have a bug here. I would suggest removing the skip-trim entry from the 'argv' variable in filter-argv-params.js (like it's done for the help argument), depending on the availability of '--skip-trim' in the process.argv.slice(2) array.

So if one used the '--skip-trim false' argument on the command-line, it would stay in 'argv' and overwrite the value set in the profile file. If it's not used, icongenie would either use the value in the profile file or it would simply not skip trimming, which is the default behavior anyway.

I hope that makes sense.

This seems to work (between the hash character comments):

[lib/utils/filter-argv-params.js](https://github.com/quasarframework/quasar/blob/bd35cdffd6f5a5b4c1be33f4ba1d587b39da1379/icongenie/lib/utils/filter-argv-params.js):
```
module.exports = function filterArgvParams(argv) {
  const params = {};

  Object.keys(argv).forEach((key) => {
    if (key.length > 1 && key !== "help") {

      // #### ignore skip-trim if not used as command-line argument
      if (key === 'skip-trim' && !process.argv.slice(2).includes('--skip-trim')) {
        return
      }
      // #### end

      // kebab to camel case
      const prop = key.replace(/(-\w)/g, (m) => m[1].toUpperCase());

      params[prop] = argv[key];
    }
  });

  return params;
};

```

A workaround is to use the '--skip-trim' argument when using icongenie, even if it was set in the profile file.

**Comment by rstoenescu** [comment]
Fix will be available in IconGenie v2.3.2.
